projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-send-email-preview.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * Handles the sending of email previews via the WordPress.com REST API.
+ *
+ * @package automattic/jetpack
+ */
+
+use Automattic\Jetpack\Connection\Manager;
+use Automattic\Jetpack\Status\Host;
+
+require_once __DIR__ . '/trait-wpcom-rest-api-proxy-request-trait.php';
+
+/**
+ * Class WPCOM_REST_API_V2_Endpoint_Send_Email_Preview
+ * Handles the sending of email previews via the WordPress.com REST API
+ */
+class WPCOM_REST_API_V2_Endpoint_Send_Email_Preview extends WP_REST_Controller {
+
+	use WPCOM_REST_API_Proxy_Request_Trait;
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->post_type                       = 'post';
```

Comment:
This looks like it might be left over from a copy/paste?

Reply:
from a rebase, good catch